prow/plugins/updateconfig/README.md

```diff
@@ -21,6 +21,10 @@ config_updater:
     path/to/some/other/thing2:
       name: thing2-config
       namespace: otherNamespace
+      # specify the cluster that the configmap targets
+      # which requies that either --build-cluster or --kubeconfig arg is enabled for Hook
+      # https://github.com/kubernetes/test-infra/blob/master/prow/getting_started_deploy.md#run-test-pods-in-different-clusters
+      cluster: default
```

Comment:
Why should a user ever set `cluster: default`? They should not set this field unless they want the configmap to _only_ be created in a non-default cluster

Reply:
Done.